### PR TITLE
fix: error handling for tput

### PR DESCRIPTION
**Pull request by varlogtim**

### CODE REVIEW

**Comment by varlogtim** [comment]
Hey @rb-determined-ai The error message about `conda` I jammed in there might be excessive, but we should at least fail with some more info when there is tput error.

File: devcluster/devcluster.py [hunk 1]
```diff
@@ -10,12 +10,28 @@
 _save_cursor = None
 
 
+def tput(args):
```
Comment (rb-determined-ai) [inline devcluster/devcluster.py:13]: how about:
```python
def tput(*args):
    cmd = ["tput"] + args
    p = subprocess.Popen(cmd, ...)
```

File: devcluster/devcluster.py [hunk 2]
```diff
@@ -10,12 +10,28 @@
...
+
+    if p.returncode != 0:
+        err_str = err.decode('utf-8').strip()
+        err_msg = str(f"'{' '.join(['tput', *args])}' "
+                      f"exited non-zero({p.returncode}):\n"
+                      f"stdout: '{out.decode('utf-8').strip()}'\nstderr: '{err_str}'\n")
+        if 'unknown terminal' in err_str:
+            err_msg += str("If using a conda environment you may need to copy "
+                           "your '/usr/share/terminfo/*/<term_name>' into "
+                           "'/path/to/conda/envs/<ENV>/share/terminfo/*/<term_name>")
```
Comment (rb-determined-ai) [inline devcluster/devcluster.py:25]: Can we suggest that `conda` users `export TERMINFO=/usr/share/terminfo` instead?  That works for urxvt on my machine.

**Review by rb-determined-ai — COMMENTED** [review]
Would you mind running `make fmt`?  We don't have ci set up yet but if we did `make check` would fail.

File: devcluster/devcluster.py (hunk 1 above)
Comment (varlogtim) [reply devcluster/devcluster.py]: Done. Although, `*args` is a tuple so we can't concatenate.

**Comment by rb-determined-ai** [comment]
Would you mind running make fmt? We don't have ci set up yet but if we did make check would fail.

**Review by rb-determined-ai — APPROVED** [review]

**Pull request merged**